documentation/src/main/asciidoc/topics/ref_memcached_client_interoperability.adoc

```diff
@@ -0,0 +1,22 @@
+[id='memcached_interoperability-{context}']
+= Interoperability with Memcached Clients
+The {brandname} Server Memcached endpoint is deprecated and disabled by
+default. If you enable the Memcached endpoint, you should use a text-based
+storage format for data. Memcached clients do not require any configuration to
+handle text-based formats.
+
+Memcached text clients can handle String-based keys and `byte[]` values but
+cannot negotiate data types with {brandname}. These clients do not offer much
+flexibility when handling data formats because of the protocol definition.
+
+[NOTE]
+====
+If you enable the Memcached endpoint, you should not use REST or Hot Rod
+clients to interoperate on the same data as Memcached clients.
+====
```

Comment:
@gustavonalle this note came from @wfink I think the intent of the note here is centered around security so I've updated to make that meaning a little more clear.

Reply:
Ok, probably because the memcached is not secure by default. So yeah, it's a bad idea, but there's a conflict of interest here: we are documenting the interop of memcached and right after telling user "don't do it" :), and we are marking it as "deprecated" but still maintaining it since at least 2 majors.
@tristantarrant Why don't we simply remove it, as it is a liability?